spec/workflow/client_spec.rb

```diff
@@ -329,9 +329,17 @@
       end
     end
 
-    context 'when it fails for any reason' do
+    context 'when the status is not found' do
       let(:status) { 404 }
 
+      it 'throws the missing workflow exception' do
+        expect { subject }.to raise_error Dor::MissingWorkflow
+      end
+    end
+
+    context 'when it fails for any reason' do
```

Comment:
when it fails with status other than 404?

Reply:
change made. thanks!